src/index.ts

```diff
@@ -0,0 +1,9 @@
+// string
+export * from "./string/words";
+export * from "./string/camelCase";
+export * from "./string/snakeCase";
+
+// object
+export * from "./object/keysCamelCase";
```

Comment:
why not to make entry for `object` (`string` too)
```js
// src/object/index.ts
export * "./keysCamelCase";
```
and 
```js
// src/index.ts
export * from "./object"
```

Reply:
yep Im considering it too